### PR TITLE
feat: add news grants category (usability/devEx)

### LINKED ISSUE #6948: Add UX/usability and DevEx as Gitcoin Grants categories
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As someone with a Gitcoin Grants project, I wish UX/usability AND DevEx were categories.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
My project doesn't fit well into the existing categories.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
Only other categories are available to choose from.

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
UX/usability and DevEx should be options.

### Definition of Done

These show up as options.

### Data Requirements / Definition of Success

Lots of projects tag themselves with this category and lots of users click on this category to sort projects.

### Additional Information

Screen Shot 2020-06-22 at 11 24 33 AM

### PR DESCRIPTION
**Pull request by developerfred**
##### Description
I changed the Grants model to add two new categories

##### Refers/Fixes
add usability and DevEx
 closes #6948

##### Testing
- testing movie: https://share.getcloudapp.com/JruL1JDb

Screen Shot 2020-06-22 at 17 12 35

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@developerfred Could you throw in a video of showing a grant created with the subcategory and the search filter working on the grants landing page for the newly added sub categories

**Comment by danlipert** [comment]
@developerfred looks like travis is failing:

=========================== short test summary info ============================
FAIL app/grants/tests/test_views.py::GrantsViewResponsesTests::test_fetching_grant_categories_from_api
Results (86.62s):
     313 passed
       1 failed
         - app/grants/tests/test_views.py:17 GrantsViewResponsesTests.test_fetching_grant_categories_from_api
The command "pytest -p no:ethereum -p no:warnings" exited with 1.
7.41s$ codecov

**Comment by developerfred** [comment]
@thelostone-mc  sure! @danlipert I will fixed test case

File: app/grants/tests/test_views.py [hunk 1]
```diff
@@ -27,12 +27,14 @@ def test_fetching_grant_categories_from_api(self):
                 ['community',5],
                 ['eth2.0',6],
                 ['eth1.x',7],
-                ['twitter', 8],
-                ['reddit', 9],
-                ['blog', 10],
-                ['notes', 11],
-                ['COVID19 research',12],
-                ['COVID19 response',13]
+                ['devEx', 8],
```
Comment (danlipert) [inline app/grants/tests/test_views.py:30]: what do these numbers mean that are in this response? if they are meaningful, we should probably append the new categories to the end of this list, right?

Comment (developerfred) [reply app/grants/tests/test_views.py:30]: right!

Comment (developerfred) [reply app/grants/tests/test_views.py:30]: I will change!

Comment (developerfred) [reply app/grants/tests/test_views.py]: @danlipert   The API returns according to the categories, so DevEx and Usability is above twitter. The order is Technology, Community, Health

**Comment by danlipert** [comment]
@developerfred ah that makes sense! thanks for the info

**Pull request merged**